youtube_dl/YoutubeDL.py

```diff
@@ -1839,7 +1840,12 @@ def ensure_dir_exists(path):
         if not self.params.get('skip_download', False):
             try:
                 def dl(name, info):
-                    fd = get_suitable_downloader(info, self.params)(self, self.params)
+                    if 'heartbeat_url' in info:
+                        print("use HTTPHB")
```

Comment:
Don't use print() directly as -j will be broken. Use to_screen or related functions instead.